mapie/tests/test_classification.py

```diff
@@ -1274,6 +1387,31 @@ def test_toy_dataset_predictions(strategy: str) -> None:
     )
 
 
+@pytest.mark.parametrize("strategy", [*STRATEGIES_BINARY])
```

Comment:
Do we check that prefit and split are the same? Additionally, do we add check of binary on other functions by just adding the parameters?

Reply:
- Do we check that prefit and split are the same?
I don't know, but that's not what this PR is about (in the sense that this test is simply a copy and paste of the multi-class test above). It might be interesting to add it to the questions if that's the case :)
- Do we add check of binary on other functions by just adding the parameters?
If this test works for binary, then by extension, all other tests on multi-class will work for binary.

Reply:
- We do check for other multi-class problems that the split and prefit are the same.
- I don't think we need to check that multi-class works in the use-case of binary. But that the tests that are done for multi-class, all that can be, should be replicated for the binary use-case.